File: Java/Maven/src/main/java/com/COMP3004/Maven/Story/tournamentCard.java
package com.COMP3004.Maven.Story;

import java.io.Serializable;

public class tournamentCard extends storyCard implements Serializable {

	private static final long serialVersionUID = 1L;
	// Properties
	int bonusShields;

	// Constructor
	public tournamentCard(String name, int bS) {
		super("TOURNAMENT", name);
		bonusShields = bS;
		behaviour = new tournamentBehaviour(this);
	}

	public int getShields() {
		return bonusShields;
	}

	@Override
	public String toString() {
		return "tournamentCard [bonusShields=" + bonusShields + "]";
	}
	
}

File: Unity/Assets/Scripts/Classes/Deck/Card/Story/tournamentCard.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class tournamentCard : storyCard {

    private int bonusShields;

    public tournamentCard(string name, int bS) : base(name, "TOURNAMENT")
    {
        bonusShields = bS;
        this.playBehavior = new playTournamentBehaviour(this);
    }

    public int getBonusShields() { return bonusShields; }
}
